cluster/saltbase/BUILD

```diff
@@ -0,0 +1,87 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar")
+
+filegroup(
+    name = "package-srcs",
+    srcs = glob(["**"]),
+    tags = ["automanaged"],
+    visibility = ["//visibility:private"],
+)
+
+filegroup(
+    name = "all-srcs",
+    srcs = [":package-srcs"],
+    tags = ["automanaged"],
+)
+
+# TODO(#3579): This is a temporary hack. It gathers up the yaml,
```

Comment:
I copied the comment from [here](https://github.com/kubernetes/kubernetes/blob/master/build/lib/release.sh#L338-L344). I'm still not sure if it's necessary or not.